docs/en/start/spec.md

```diff
@@ -66,6 +70,66 @@ More details can be found in [Using Pre-Processors](../configurations/pre-proces
 
 - By default, contents will be extracted and dynamically inserted into the document's `<head>` as an actual `<style>` tag using `style-loader`. It's also possible to [configure Webpack so that all styles in all components are extracted into a single CSS file](../configurations/extract-css.md).
 
+### Custom Blocks
+
+Additional custom blocks can be included in a `*.vue` file for any project specific needs.  `vue-loader` will use the tag name to look up which webpack loaders should be applied to the contents of the section.  The webpack loaders should be specified in the `loaders` hash of the `vue` section of the webpack configuration in the same way that languages are specified for the standard sections of the file.  See [Advanced Loader Configuration](../configurations/advanced.md).
```

Comment:
Sorry, I'm not clear on what you're asking for here.  Is it 1 space between sentences instead of 2?  If so, I just changed that and will push.